From 9b2dbd7d4219ca977edf9bc115d56b2e946f2438 Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Fri, 2 Mar 2012 16:28:21 +0100
Subject: [PATCH 10/12] Add missing documentation for qemu-img -p

RH-Author: Kevin Wolf <kwolf@redhat.com>
Message-id: <1330705703-8967-3-git-send-email-kwolf@redhat.com>
Patchwork-id: 37893
O-Subject: [RHEL-6.3 qemu-kvm PATCH 2/4] Add missing documentation for qemu-img -p
Bugzilla: 725748
RH-Acked-by: Ademar de Souza Reis Jr. <areis@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

From: Jes Sorensen <Jes.Sorensen@redhat.com>

Bugzilla: 725748

Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
Signed-off-by: Kevin Wolf <kwolf@redhat.com>
(cherry picked from commit aaf55b4795d95d87353a08710f237f88d81a3c35)

We already have one part of this upstream commit, this is the remaining
change that went into RHEL in the wrong order.

Conflicts:

	qemu-img-cmds.hx
	qemu-img.texi

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
---
 qemu-img.texi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-img.texi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/qemu-img.texi b/qemu-img.texi
index ce53411..9a670ab 100644
--- a/qemu-img.texi
+++ b/qemu-img.texi
@@ -121,7 +121,7 @@ they are displayed too.
 
 List, apply, create or delete snapshots in image @var{filename}.
 
-@item rebase [-f @var{fmt}] [-u] -b @var{backing_file} [-F @var{backing_fmt}] @var{filename}
+@item rebase [-f @var{fmt}] [-p] [-u] -b @var{backing_file} [-F @var{backing_fmt}] @var{filename}
 
 Changes the backing file of an image. Only the formats @code{qcow2} and
 @code{qed} support changing the backing file.
-- 
1.7.7.6

